browser/ui/playlist/desktop_playlist_player.cc

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/playlist/desktop_playlist_player.h"
+
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "brave/browser/playlist/playlist_service_factory.h"
+#include "brave/components/playlist/playlist_service.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/browser/ui/scoped_tabbed_browser_displayer.h"
+#include "content/public/browser/browser_context.h"
+#include "content/public/browser/page_navigator.h"
+#include "net/base/filename_util.h"
+
+namespace playlist {
+
+DesktopPlaylistPlayer::DesktopPlaylistPlayer(content::BrowserContext* context)
+    : context_(context) {}
+
+DesktopPlaylistPlayer::~DesktopPlaylistPlayer() {}
+
+void DesktopPlaylistPlayer::Play(const std::string& id) {
+  auto* service =
+      PlaylistServiceFactory::GetInstance()->GetForBrowserContext(context_);
+  if (!service)
+    return;
+
+  auto html_file_path = service->GetPlaylistItemDirPath(id).Append(
+      FILE_PATH_LITERAL("index.html"));
```

Comment:
seems like this should be a constant so it doesn't get out of sync with PlaylistService